<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunityFundDisplay</class>
 <widget class="QWidget" name="CommunityFundDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutProposal">
     <item>
      <widget class="QLabel" name="title">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutStatus">
       <item>
        <widget class="QLabel" name="labelTitleStatus">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelStatus">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutRequested">
       <item>
        <widget class="QLabel" name="labelTitleRequested">
         <property name="text">
          <string>NAV Requested:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRequested">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutDuration">
       <item>
        <widget class="QLabel" name="labelTitleDuration">
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDuration">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutButtons">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDetails">
         <property name="text">
          <string>View Detail</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
